test/e2e/send.feature

```diff
@@ -3,6 +3,9 @@ Feature: Send dialog
   Scenario: should allow to send when enough funds and correct address form
     Given I'm logged in as "genesis"
     And I fill in "1" to "amount" field
+    Then I should see "convertor" element
+    Given I wait 5 seconds
+    Then I should see "convertElem" element with text matching coverter price
```

Comment:
In e2e tests, we have a convention of kebap-case class names with dashes replaced from spaces. So here it would be "convert elem", in the step definition, you already have `replace(/ /g, '-')`, so in the converter component you can have `convert-elem` className. Of course, you'll need to update the className also in unit tests.

A more descriptive className would IMO be `converted-price`.